examples/server/server.c

```diff
@@ -1118,7 +1134,8 @@ THREAD_RETURN WOLFSSL_THREAD server_test(void* args)
 #ifdef WOLFSSL_WOLFSENTRY_HOOKS
     wolfsentry_errcode_t wolfsentry_ret;
 #endif
-
+    int    minVersion = SERVER_INVALID_VERSION;
```

Comment:
If you have minVersion to an invalid value, why not check that instead of using setMinVersion?

Reply:
That makes sense. Will update Client too